src/tasks/setupDNS/darwin.js

```diff
@@ -52,9 +51,4 @@ export default {
         task: () => execa('killall', ['-HUP', 'mDNSResponder']),
       },
     ]),
-  skip: async () => {
-    const exists = await fileExists(resolverPath);
```

Comment:
is this so we can keep changing the port if we want to?

Reply:
Ya, it's just overwriting the file every time now. I figure the overhead isn't too bad, and it's less code and complexity than checking if the port equals the previous one in the file before.

Reply:
this will require elevated access each time.  I guess thats ok, but not ideal.

Reply:
could be an update at a later time.